### PR TITLE
Exams by Vlad Tokar IM21

### PR DESCRIPTION
**Pull request by Vladyulik**
Refactored quotes.js and words.js

### CODE REVIEW

File: Tasks/quotes.js [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
-          open = false;
-        }
-      }
-    } else {
-      if (c !== '"') {
-        for (i of c) {
-          res.push(i);
-        }
-      }
+const quotes = (string) => {
```
Comment (kovaliovev) [inline Tasks/quotes.js:5]: It would be better to name function as `changeQuotes` for clarity.

**Review by kovaliovev — CHANGES_REQUESTED** [review]

File: Tasks/words.js [hunk 2]
```diff
@@ -1,41 +1,14 @@
...
-        if (flag === true) {
-          flag = true;
-        } else {
-          flag = true;
-        }
-      }
-    }
+'use strict';
+
+const words = (string) => {
```
Comment (sonakrrr) [inline Tasks/words.js:5]: `countWords` would be a better name for the function

**Review by sonakrrr — CHANGES_REQUESTED** [review]

**Review by kovaliovev — APPROVED** [review]
Great work!

**Review by sonakrrr — APPROVED** [review]
you did good!

File: Tasks/quotes.js [hunk 3]
```diff
@@ -1,30 +1,18 @@
...
-        for (i of c) {
-          res.push(i);
-        }
-      }
+const changeQuotes = (string) => {
+  const result = [];
+  let open = false;
+  for (let char of string) {
+    if (char === '"') {
+      !open ? char = '«' : char = '»';
```
Comment (tshemsedinov) [inline Tasks/quotes.js:10]: Incorrect usage of ternary semantics:
```suggestion
      char = !open ? '«' : '»';
```

**Comment by Vladyulik** [comment]
fixed all the problems 
https://github.com/HowProgrammingWorks/Exams/commit/d0a4eb4cdf9d20427c06fdaadafc6d24e55ce899

24 січня 2023, 19:16:57, від "Timur Shemsedinov" ***@***.***>:


@tshemsedinov commented on this pull request.
In Tasks/quotes.js:
> -          res.push('»');
-          open = false;
-        }
-      }
-    } else {
-      if (c !== '"') {
-        for (i of c) {
-          res.push(i);
-        }
-      }
+const changeQuotes = (string) => {
+  const result = [];
+  let open = false;
+  for (let char of string) {
+    if (char === '"') {
+      !open ? char = '«' : char = '»';
Incorrect usage of ternary semantics:
⬇️ Suggested change -      !open ? char = '«' : char = '»';
+      char = !open ? '«' : '»';

**Pull request closed without merging**